metricbeat/beater/metricbeat.go

```diff
@@ -1,8 +1,11 @@
 /*
+Package beater provides the implementation of the libbeat Beater interface for
```

Comment:
I was first confused by "Package beater". It is definitively correct but perhaps use "The package beater" or "This package provides".